packages/contracts-node/README.md

```diff
@@ -0,0 +1,71 @@
+# @uma/contracts-node
```

Comment:
@daywiss @nicholaspai @chrismaree PTAL at the node-centric README. This should be slightly better than the old import style. One big advantage is that the typechain types no longer require you to dig into the package directories as we've done in the past. You should be able to import them directly from the `@uma/contracts-node` package.

There is also no versioning anymore. To get versioning, you'll need to alias old versions of core or this package.